js/src/wp-seo-featured-image.js

```diff
@@ -160,6 +161,30 @@ import a11ySpeak from "a11y-speak";
 		if ( "undefined" !== typeof $featuredImageElement.prop( "src" ) ) {
 			featuredImagePlugin.setFeaturedImage( $( "#set-post-thumbnail " ).html() );
 		}
+
+		// Fallback for Gutenberg, as the featured image id does not exist there.
+		let imageData;
+		if ( isGutenbergDataAvailable() ) {
+			wp.data.subscribe( () => {
+				const featuredImageId = wp.data.select( "core/editor" ).getEditedPostAttribute( "featured_media" );
+
+				if ( featuredImageId === undefined ) {
+					return;
+				}
+
+				// Only continue if image data has changed.
+				if ( imageData !== wp.data.select( "core" ).getMedia( featuredImageId ) ) {
```

Comment:
You're calling `wp.data.select( "core" ).getMedia( featuredImageId )` twice: Once here and once on line 178. I'd suggest placing it in a variable above this `if`.